class/practice_5.1/include/Helpers.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <array>
+#include <vector>
+
+class MegaData {
+public:
+  MegaData();
+
+  void reset();
+
+  const std::array<float, 1024>& getSmallArray() const;
```

Comment:
both 512 and 1024 can be named constexpr constant. You can use constexpr as a template arguments as they are known at compile time